skimage/color/tests/test_delta_e.py

```diff
@@ -30,7 +30,7 @@ def test_ciede2000_dE(dtype, channel_axis):
     dE2 = deltaE_ciede2000(lab1, lab2, channel_axis=channel_axis)
     assert dE2.dtype == _supported_float_type(dtype)
 
-    rtol = 1e-2 if dtype == np.float32 else 1e-4
+    rtol = 1e-2 if dtype == np.float32 else 6e-3
     assert_allclose(dE2, data['dE'], rtol=rtol)
```

Comment:
Were these relative tolerances set empirically or some other way? The tests fail for only the first entry in `dE2` with numpy v1.24.0.

Reply:
pretty sure it is empirical (I think I added the 1e-2 for float32 at some point, but not sure about the original choice of 1e-4). I don't know why this particular function seems to have relatively low precision.